packages/starlight/components/Footer.astro

```diff
@@ -24,7 +24,11 @@ const prevNextLinks = getPrevNextLinks(sidebar);
         <EditLink data={entry.data} id={entry.id} {locale} />
       )
     }
-    <LastUpdated id={entry.id} {lang} {locale} />
+    {
+      config.showUpdateDate && (
```

Comment:
Based on that, I’d guess we do something like the following so that the frontmatter value takes precedence if set when deciding whether or not to show this:

```suggestion
      (entry.data.lastUpdated ?? config.lastUpdated) && (
```